application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -186,7 +189,10 @@ adminTools.dashboard.showLogs=View last n lines of log...
 adminTools.dashboard.spamPage.modal.title=Pages with more than {0} comments
 adminTools.dashboard.spamPage.modal.notFound=There are no pages with more than {0} comments.
 adminTools.dashboard.spamPage.modal.header.page=Page
-adminTools.dashboard.spamPage.modal.header.noComments=Number of comments</content>
+adminTools.dashboard.spamPage.modal.header.noComments=Number of comments
+adminTools.dashboard.healthcheck.modal.title=Flush Wiki cache action
```

Comment:
Shouldn't it be `Flush instance cache`? If this is specific to a wiki, how can I flush the cache for the other wikis?